### PR TITLE
[VGNWB-328] Export affiliation view

### PR DESCRIPTION
**Pull request by nowke**
Related FE - https://github.com/diging/vogon-web-client/pull/36

### CODE REVIEW

**Comment by tjquinn1** [comment]
please fix failing checks

**Comment by tjquinn1** [comment]
resolve the conflicts but this also did not work for me. It returns a 500 with the error on the server being:
annotations.models.RelationTemplate.DoesNotExist: RelationTemplate matching query does not exist.

**Comment by nowke** [comment]
`annotations.models.RelationTemplate.DoesNotExist: RelationTemplate` requires a template with `name=Affiliation`.

**Comment by tjquinn1** [comment]
see front end issues

**Comment by nowke** [comment]
Reopening as discussed.

**Comment by tjquinn1** [comment]
So when I run the migrations for this the first one runs but the second one fails.

Screen Shot 2020-10-14 at 10 08 26 AM

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
I can't test this until amphora connection issue resolved

File: annotations/models.py [hunk 1]
```diff
@@ -453,8 +444,7 @@ def _re(text):
     def __unicode__(self):
         return self.title
 
-# TODO: remove this model, as it is no longer used (in favor of the repository
-#  module).
+
```
Comment (tjquinn1) [inline annotations/models.py]: remove the extra white space

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

File: annotations/models.py (hunk 1 above)
Comment (nowke) [reply annotations/models.py]: Done

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
When the relations are exported it throws a 404. The logs say:

        raise exc
      File "/usr/local/lib/python3.7/site-packages/rest_framework/views.py", line 494, in dispatch
        response = handler(request, *args, **kwargs)
      File "/usr/src/app/host/annotations/views/project_views.py", line 80, in export_affiliations
        template = RelationTemplate.objects.get(name="Affiliation")
      File "/usr/local/lib/python3.7/site-packages/django/db/models/manager.py", line 82, in manager_method
        return getattr(self.get_queryset(), name)(*args, **kwargs)
      File "/usr/local/lib/python3.7/site-packages/django/db/models/query.py", line 408, in get
        self.model._meta.object_name
    annotations.models.RelationTemplate.DoesNotExist: RelationTemplate matching query does not exist.

I checked the admin and the template exists and the relation set is using the correct template

**Comment by nowke** [comment]
Changed method of finding Affiliation template usnig id, instead of name. This is configurable from `settings.py`. Indicated the change in https://diging.atlassian.net/wiki/spaces/VGNWB/pages/2086273044/Vogon+Initial+Setup

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Fix conflicts

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Comment by tjquinn1** [comment]
Make it so, Jenkins.

**Pull request merged**